tests/widget/inline/widget-inline.register.test.js

```diff
@@ -7,7 +7,7 @@ test('Inline registration widget', async ({ page }) => {
   await page.fill('text=Username', uuid());
   await page.fill('text=First Name', 'Demo');
   await page.fill('text=Last Name', 'User');
-  await page.fill('text=Email Address', 'demo@user.com');
+  await page.fill('text=Email Address', 'test@auto.com');
```

Comment:
Changing the email used for auto-generated users to more easily manage the deletion of them afterwards. Plus, when using "forgot password" or "forgot username" if more than one user has the email you enter, the journey just fails with a 401.